src/NHibernate.Test/NHSpecificTest/DtcFailures/DtcFailuresFixture.cs

```diff
@@ -87,8 +92,8 @@ public void WillNotCrashOnDtcPrepareFailure()
 		[Test]
 		public void Can_roll_back_transaction()
 		{
-			if (Dialect is FirebirdDialect)
-				Assert.Ignore("Firebird driver does not support distributed transactions");
+			/*if (Dialect is FirebirdDialect)
+				Assert.Ignore("Firebird driver does not support distributed transactions");*/
```

Comment:
In my local test Firebird supports them, so just letting them run to test with teamcity too.